CMakeLists.txt

```diff
@@ -99,6 +99,12 @@ if(NOT DAEMON_EXTERNAL_APP)
         option(BUILD_GAME_NACL "Build the NaCl gamelogic modules, required to host mods." OFF)
         option(BUILD_GAME_NACL_NEXE "Build the NaCl \"nexe\" files from the \"pexe\" files." ON)
         mark_as_advanced(BUILD_GAME_NACL_NEXE)
+        option(BUILD_GAME_NACL_NEXE_I686 "Build the NaCl \"nexe\" files for the i686 architecture" ON)
+        mark_as_advanced(BUILD_GAME_NACL_NEXE_I686)
+        option(BUILD_GAME_NACL_NEXE_AMD64 "Build the NaCl \"nexe\" files for the amd64 architecture" ON)
+        mark_as_advanced(BUILD_GAME_NACL_NEXE_AMD64)
+        option(BUILD_GAME_NACL_NEXE_ARMHF "Build the NaCl \"nexe\" files for the armhf architecture" ON)
```

Comment:
default to off?

Reply:
I think defaulting to on is good, but that we shouldn't build nacl things by default. LGTM

Reply:
Right now `BUILD_GAME_NACL` build is `OFF` by default so any depending var can be `ON` they won't have any effect until `BUILD_GAME_NACL` is turned to `ON`.